syntax = "proto3";

package request_models;

message ReloadWeaponRequestModel {
	int32 player_id = 1;
}